assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -19,13 +19,27 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-idea-hub__dashboard-cta {
+		align-items: center;
 		background-color: $c-white;
-		display: flex;
+		height: 100%;
+		padding: $grid-gap-desktop;
+
+		@media (min-width: $bp-tablet) {
+			display: flex;
+		}
+
+		@media (min-width: $bp-desktop) {
+			padding: 48px;
```

Comment:
Set padding to `24px` here via the `$grid-gap-desktop` variable. Thus we will have `24px` padding from `googlesitekit-widget__body` and `24px` here which totals to `48px` 😁